paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/ImageUtils.kt

```diff
@@ -228,8 +229,20 @@ internal object ImageUtils {
 
     var sourceWidth = source.width
     var sourceHeight = source.height
-    val destWidth = Math.max(1, (xScale * sourceWidth).toInt())
-    val destHeight = Math.max(1, (yScale * sourceHeight).toInt())
+
+    fun Double.roundAppropriately(): Int {
+      return if (xScale > 1f) {
+        roundToInt()
+      } else {
+        toInt()
+      }
+    }
+
+    // When rounding up use round to nearest to fix wear images.
+    // Keep current logic when scaling down, truncating using a floor.
+    val destWidth = (xScale * sourceWidth).roundAppropriately().coerceAtLeast(1)
+    val destHeight = (yScale * sourceHeight).roundAppropriately().coerceAtLeast(1)
+
```

Comment:
i don't think this is necessary?  What happens if you simply copy the golden image from:

`ROOT/paparazzi/paparazzi-gradle-plugin/src/test/projects/compose-wear/build/reports/paparazzi/images/331377f7bc34cf4d9c0c3624164e314577328244.png
`

into:

`ROOT/paparazzi/paparazzi/paparazzi-gradle-plugin/src/test/projects/compose-wear/build/reports/paparazzi/images/compose_wear.png`

Reply:
Yep - done.